graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/mapping/FieldMappingApi.java

```diff
@@ -0,0 +1,34 @@
+package org.graylog.storage.elasticsearch7.mapping;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Streams;
+import org.graylog.shaded.elasticsearch7.org.elasticsearch.client.Request;
+import org.graylog.shaded.elasticsearch7.org.elasticsearch.client.Response;
+import org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class FieldMappingApi {
+    private final ObjectMapper objectMapper;
+
+    @Inject
+    public FieldMappingApi(ObjectMapper objectMapper) {
```

Comment:
Let's inject `RestHighLevelClient` in the constructor, as we do for the other `*Api` wrappers.

Reply:
:heavy_check_mark: